controller/server_test.go

```diff
@@ -413,13 +412,13 @@ func TestServer_TaskUpdate(t *testing.T) {
 		assert.Equal(t, "", plan, "run now does not return plan info")
 		assert.False(t, changed, "run now does not return plan info")
 
-		events := ctrl.store.Read(taskName)
+		events := ctrl.state.GetTaskEvents(taskName)
 		assert.Len(t, events, 1)
 	})
 }
 
 // mockDriver sets up a mock driver with the happy path for all methods
-func mockDriver(ctx context.Context, d *mocks.Driver, task *driver.Task) {
+func mockDriver(ctx context.Context, d *mocksD.Driver, task *driver.Task) {
```

Comment:
Context fyi - mocks.Driver was imported twice in this file which was giving me a linting error. I removed `mocks` in the imports and kept `mocksD`:

```diff
-	mocks "github.com/hashicorp/consul-terraform-sync/mocks/driver"
 	mocksD "github.com/hashicorp/consul-terraform-sync/mocks/driver"
```